Content.Shared/Devour/Components/DevourerComponent.cs

```diff
@@ -71,6 +72,18 @@ public sealed partial class DevourerComponent : Component
     [ViewVariables(VVAccess.ReadWrite), DataField("healRate")]
     public float HealRate = 15f;
 
+    /// <summary>
+    /// The chemical ID's injected into the victim upon being devoured.
+    /// </summary>
+    [DataField("stomachChemicals", customTypeSerializer: typeof(PrototypeIdListSerializer<ReagentPrototype>))]
+    public List<string> StomachChemicals = new();
```

Comment:
Please update to modern codebase standards i.e. no datafield string and using protoid.

Reply:
I'll do a reread of the new conventions before I will make any new PR's